### PR TITLE
Fonts and images bugs fixed

### PR DESCRIPTION
**Pull request by Aleksey-Bortnik**
1) .gitignore 
add empty line at the end of file

2) REMOVE dist/ folder from the repository, add it to .gitignore

3) Remove ',' between classes

4) You can use
<ul> <li>...</li> ... </ul> for dots

5) Why did you add 'id' here? (HTML 263 <div id="registration" class="footer__input-container">

6) Use scss useful features (&)

7)Try to not use concretize sizes for buttons, use paddings

8)Why margin-left 5% here? (design.scss   .designers__section-image { margin-left: 5%;

9) Remove unnecessary comments

10) Add common class for these sections to avoid classes listing

11) with @include you still add the code to all places mixin is used
all font properties in CSS are inherited so that's why base font-family and font-size are usually set on body selector
then font properties are specified for other element if they have different styling
no need to add the same styles everywhere
 
12) use <i> tag for icons

13) <h2>
<span class="marketers__section-text">Optimization for</span>
Marketers
</h2>
all the text is header by meaning and part of it styled differently

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      <article>
+        <img class="strategies-section__image-time" src="./assets/img/strategies/strategies-time.png"
+          alt="strategies content picture" />
+        <p>By <span class="strategies-section__accent">Abdullah |</span> 03 March 2019</p>
+        <h4 class="strategies-section__title">Success Steps For Your Personal Or Business Life</h4>
+      </article>
+    </div>
+
+    <ul class=" strategies-section__carousell">
+
```
Comment (ElizavetaSelivanova) [inline index.html]: No need to separate here - ul = its one logical block

File: index.html [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      <li class="circle"></li>
+      <li class="circle"></li>
+
+    </ul>
+
+  </section>
+  <footer class="footer block__container">
+    <div class="footer__container">
+      <div class="footer__logo-container">
+        <img class="footer-logo" src="./assets/img/logo/logo.png" alt="site logo">
```
Comment (ElizavetaSelivanova) [inline index.html]: `/>` =)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+      <div class="footer__input-container">
+        <h4 class="footer__description footer__description-input">Knowing you're always on the best energy deal.</h4>
+        <input class="footer__input" type="number" placeholder="Enter your phone Number">
+        <button class="footer__input-button">Sing up Now</button>
+      </div>
+    </div>
+  </footer>
+</body>
+
+</html>
```
Comment (ElizavetaSelivanova) [inline index.html:277]: no empty line

File: scss/components/_strategies.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.circle {
+  list-style-type: none;
+  text-align: center;
+  width: 16px;
+  height: 16px;
+  border: 1px sold #efefef;
+  border-radius: 50%;
+  background-color: #ddd4d4;
```
Comment (ElizavetaSelivanova) [inline scss/components/_strategies.scss]: Use variables

File: scss/main.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+@import "./components/design";
+
+@import "./components/product";
+
+@import "./components/marketers";
+
+@import "./components/strategies";
+
+@import "./components/footer";
```
Comment (ElizavetaSelivanova) [inline scss/main.scss:21]: Separate imports and common styles

**Review by ElizavetaSelivanova — CHANGES_REQUESTED** [review]
Great Job!)

**Comment by ElizavetaSelivanova** [comment]
And also, resolve conflicts, please )

File: scss/components/_header.scss [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  text-decoration: 0;
+  margin: 0 15px;
+  min-width: 66px;
+}
+
+.nav__btn-container {
+  display: flex;
+}
+
+.nav__container-button-in {
```
Comment (irina-chernik) [inline scss/components/_header.scss]: be consistent with naming
.nav__btn-container
.nav__container-button-in
btn vs button

what does button-in mean? why _in_?

File: scss/main.scss [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  @include title-formatter(16px, 26px);
+  margin: 12px 0;
+}
+
+.blocks-title {
+  @include title-formatter (16px, 54px);
+}
+
+.common-text-index {
+@include title-formatter(16px, 24px);
```
Comment (irina-chernik) [inline scss/main.scss]: add indent for styles

File: scss/components/_header.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  margin: 30px 0 40px 0;
+  width: 1400px;
```
Comment (irina-chernik) [inline scss/components/_header.scss]: don't use fixed width
use max-width

File: scss/components/_footer.scss [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+  &__input {
+    border: 3px solid $border-color;
+    border-radius: 3px;
+    width: 325px;
+    height: 48px;
+    color: $border-color;
+  }
+
+  &__input-button {
+    width: 131px;
```
Comment (irina-chernik) [inline scss/components/_footer.scss]: do not use fixed width

File: index.html [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+    <div class="tools-section__title-wrap">
+      <h2 class="tools-section__title primary-title">We design tools to unveil
+        your Superpowers
+      </h2>
+    </div>
+
+    <div class="tools-section__cards-container ">
+
+      <article class="tools-section__cards-indentation">
+        <div class="tools-section__card-click"></div>
```
Comment (irina-chernik) [inline index.html:73]: use tag i for icons

**Review by irina-chernik — CHANGES_REQUESTED** [review]

**Pull request closed without merging**